.gitignore

```diff
@@ -39,3 +39,12 @@ x-pack/dockerlogbeat/temproot.tar
 *.test
 *.prof
 *.pyc
+
+# agent
```

Comment:
I would suggest to make this gitignore file more local go `x-pack/agent` to not "pollute" the global file.